openmc/mesh.py

```diff
@@ -1223,12 +1225,12 @@ class CylindricalMesh(StructuredMesh):
 
     def __init__(
         self,
+        origin: Tuple[float] = (0., 0., 0.),
+        r_grid: Sequence[float] = (0, 10),
```

Comment:
Again, I don't think it makes sense to assign a default `r_grid` and would prefer that this be a required argument with no default. In this case, it seems cleaner to have the three grids first, then origin, then everything else.